config/200-clusterroles.yaml

```diff
@@ -304,6 +306,7 @@ rules:
   - azurequeuestoragesources/finalizers
   - azureservicebusqueuesources/finalizers
   - azureservicebustopicsources/finalizers
+  - cloudeventssources/finalizers
   - googlecloudauditlogssources/finalizers
```

Comment:
This source doesn't implement any finalizer.

Reply:
Sorry, scratch that comment 🤦  I forgot we had to allow updates to finalizers because of that non-default admission controller used in OpenShift.